mfthomps/CyberCIEGE-port-to-Unity3d
# added missing server rack and table

added serverrack and table.

to test,
- notice the serverrack2 under MayaArt/objects/serverrack/scenes looks like the serverrack in the original 
- notice the table under Mayary/objects/table looks more like the table in the original
